scripts/build.js

```diff
@@ -92,6 +92,7 @@ async function build(target) {
     'rollup',
     [
       '-c',
+      '--bundleConfigAsCjs',
```

Comment:
Could we change the config file to ES module and remove this option?

Reply:
rename `rollup.config.js` to `rollup.config.mjs`, and change commonjs `require` to es `import {} from '...'`

Reply:
Working on it.

Reply:
> rename `rollup.config.js` to `rollup.config.mjs`, and change commonjs `require` to es `import {} from '...'`

Done